sdk/core/azure-core-tracing-opentelemetry/setup.py

```diff
@@ -44,7 +44,6 @@
         "Programming Language :: Python",
         "Programming Language :: Python :: 3 :: Only",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
```

Comment:
while we are here, might as well add in 3.11 :)

Reply:
Good call. Updated :)